docs/br/learn/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Aprenda sobre Ethereum
+meta:
+  - name: description
+    content: A set of technical and non-technical articles, guides, and resources to learn about Ethereum.
```

Comment:
These descriptions aren't necessary but if we include them we should translate them.